app/views/projects/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= @project.name %></h1>
+
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
+<p>Average Contestant Experience: <%= Contestant.avg_experience %></p>
```

Comment:
It's not conventional to call directly on the Model from within your view. Calling on the model is a job done by the controller. Alternatively, I'd prefer to see `@ project.contestants.avg_experience`